plasmidfinder/1.0/Dockerfile

```diff
@@ -0,0 +1,70 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the folowing stage. We use `app` for the production image
+FROM debian:stretch as app
+
+#https://bitbucket.org/genomicepidemiology/plasmidfinder/issues/10/version-number
+# ARG sets environment variables during the build stage
+ARG PLASMIDFINDER_VER="2.1"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+# Optional, but highly recommended
+LABEL base.image="debian:stretch"
+LABEL dockerfile.version="1"
+LABEL software="plasmidfinder"
+LABEL software.version=$PLASMIDFINDER_VER
+LABEL description="Identifies plasmids in total or partial sequenced isolates of bacteria."
+LABEL website="https://github.com/StaPH-B/docker-builds"
+LABEL license="https://github.com/StaPH-B/docker-builds/blob/master/LICENSE"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+
+
+ENV DEBIAN_FRONTEND noninteractive,Teletype
+
+# RUN executes code during the build
+# Install dependencies via apt-get or yum if using a centos or fedora base
+RUN set -xe && \
+    apt-get update -y && apt-get install -y \
+    git \
+    apt-utils \
+    wget \
+    python3-pip \
+    ncbi-blast+ \
+    gzip \
+    libz-dev
+    #rm -rf /var/cache/apt/* /var/lib/apt/lists/*
+
+# Install python dependencies
+RUN pip3 install --upgrade pip && \
+    python3 -m pip install tabulate && \
+    pip3 install -U biopython==1.73 cgecore
+
+# Install kma
+RUN git clone --branch 1.0.1 --depth 1 https://bitbucket.org/genomicepidemiology/kma.git
+RUN cd kma && make && \
+    mv kma* /bin/
+
+RUN git clone https://bitbucket.org/genomicepidemiology/plasmidfinder.git && \
```

Comment:
Instead of git clone, something like
```
wget https://bitbucket.org/genomicepidemiology/plasmidfinder/get/${PLASMIDFINDER_VER}.tar.gz
```
is better because it specifies the version

Reply:
Implemented this suggestion.